sfdc37/salesforce_types.bal

```diff
@@ -26,10 +26,8 @@ public type SalesforceError record {
 
 # Errors by HTTP or Salesforce.
 # + message - Array of string error messages
-# + cause - Array of errors
 # + salesforceErrors - Array of SalesforceError type errors
 public type SalesforceConnectorError record {
```

Comment:
Shall we remove this with Ballerina `error` since `SalesforceError` record is always `[]` and there is no usage of it?

Reply:
Here we are using SalesforceError

https://github.com/wso2-ballerina/module-salesforce/blob/master/sfdc37/salesforce_utils.bal#L109-L114

Reply:
I missed it. Then let's keep it for now.